package/ios/.swift-format

```diff
@@ -0,0 +1,11 @@
+{
+  "indentation": {
+    "spaces": 4
+  },
+  "rules": {
+    "AlwaysUseLowerCamelCase": false
```

Comment:
Just went to compare and it looks like this matches sdk-swift but not the other two - we should synchronize those at some point